api/src/AspNetCoreDemoApp/Models/User.cs

```diff
@@ -8,6 +8,15 @@ public class User : IFirestoreDocumentModel
     {
         public string Id { get; set; }
 
+        [FirestoreProperty(name: "name")]
+        public string Name { get; set; }
```

Comment:
I know it seems silly, but I think we do need this just because of the FirebaseUserModel and the way that we are able to send it over to the database.

Reply:
OK. I do understand we borrowed the user authentication portion and it might be designed a certain way. So, If you think we need it then I won't fight it.